patching/starter.py

```diff
@@ -0,0 +1,34 @@
+import argparse
+import asyncio
+
+from temporalio.client import Client
+
+# Since it's just used for typing purposes, it doesn't matter which one we
+# import
+from workflow_1_initial import MyWorkflow
+
+
+async def main():
+    parser = argparse.ArgumentParser(description="Run worker")
+    parser.add_argument("--start-workflow", help="Start workflow with this ID")
+    parser.add_argument("--query-workflow", help="Query workflow with this ID")
```

Comment:
nit: I'd make these subcommands:
https://docs.python.org/3/library/argparse.html#sub-commands

I'd also encourage using a nicer argument parser like [click](https://click.palletsprojects.com/en/8.1.x/) for samples where we don't care too much about the additional dependency.